### ISSUE TITLE
Keybase unable to start on Fedora 30, segfault

### ISSUE THREAD

**Issue by predatorian3** [keybase/keybase-issues#3851]
After updating my system and the latest stable published updates for Fedora 30 KDE Spin, I receive the following error.

```
➜  ~ dmesg | tail
[   87.469478] Keybase[3045]: segfault at 73 ip 000055c4324b6c9b sp 00007ffef5096620 error 4 in Keybase[55c42d38c000+53d3000]
[   87.469483] Code: 8b 7b 20 48 8b 07 48 83 c4 08 5b 5d ff 60 10 55 48 89 e5 41 56 53 48 89 fb 48 8b 7f 18 48 8b 07 ff 50 18 41 89 c6 48 8b 7b 20 <48> 8b 07 ff 50 18 44 01 f0 5b 41 5e 5d c3 cc cc cc cc cc cc cc 55
[  123.536681] Keybase[4132]: segfault at 73 ip 000055c737370c9b sp 00007ffebaa80790 error 4 in Keybase[55c732246000+53d3000]
[  123.536689] Code: 8b 7b 20 48 8b 07 48 83 c4 08 5b 5d ff 60 10 55 48 89 e5 41 56 53 48 89 fb 48 8b 7f 18 48 8b 07 ff 50 18 41 89 c6 48 8b 7b 20 <48> 8b 07 ff 50 18 44 01 f0 5b 41 5e 5d c3 cc cc cc cc cc cc cc 55
```

Keybase version information

```
➜  ~ dnf list keybase
Last metadata expiration check: 0:00:12 ago on Wed 18 Mar 2020 10:50:14 AM CDT.
Installed Packages
keybase.x86_64                                5.3.0.20200310205642.4f2689009b-1                                @keybase
```

My kernel version:

```
➜  ~ uname -a
Linux localhost.localdomain 5.5.9-100.fc30.x86_64 #1 SMP Thu Mar 12 14:15:32 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
➜  ~ cat << EOF      
Node Info
----------
node: $(uname -n)
system: $(uname -s)
version: $(uname -v)
release: $(uname -r)
arch: $(uname -m)

EOF
Node Info
---------- 
node: localhost.localdomain          
system: Linux
version: #1 SMP Thu Mar 12 14:15:32 UTC 2020
release: 5.5.9-100.fc30.x86_64
arch: x86_64
```

**Comment by predatorian3** [comment]
The CLI is still functioning as expected.

**Comment by heronhaye** [comment]
@predatorian3 can you try the nightly build at https://keybase.io/docs/linux-user-guide#nightly-builds?

**Comment by predatorian3** [comment]
the nightly build works

**Comment by heronhaye** [comment]
All right, we've released version 5.3.1 which should fix the issue. Please upgrade and let me know if it still doesn't work.

If you installed the nightly, you can downgrade to the release version (manually reinstall from https://keybase.io/docs/the_app/install_linux), or you can just stay on the nightly (you'll get an update for the next stable release when it comes out).

**Comment by predatorian3** [comment]
sounds good, thank you @heronhaye !
